### PR TITLE
[CON-548] feat: Added the provider guide for Sellsy Connector

### PR DESCRIPTION
**Pull request by sanjay-a01**

### CODE REVIEW

File: src/provider-guides/sellsy.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Follow the steps below to create an Sellsy app and add the Ampersand redirect URL.
+
+1. Log in to your [Sellsy](https://login.sellsy.com/login) account.
+
+2. Click **Menu**, then click **Settings**, and select **Developer Portal** in your Sellsy account.
+
+3. Click the **API V2 (beta)** tab, then click **Create API Access** and select your **API access type**: Private
+
+4. Enter the **Access name** for the Creating an API Access.
```
Comment (RajatPawar) [inline src/provider-guides/sellsy.mdx:40]: What does this mean?

File: src/provider-guides/sellsy.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+5. Enter the Ampersand redirect URL: `https://api.withampersand.com/callbacks/v1/oauth` in the **Redirect URLs** section.
+
+6. Choose the **scopes** that are applicable for your application.
+
+   [Sellsy App Creation](/images/provider-guides/sellsy_OAuth.gif)
+
+7. Click **Save**.
```
Comment (RajatPawar) [inline src/provider-guides/sellsy.mdx:48]:
```suggestion
7. Make sure to click on **Save**.
```

File: src/provider-guides/sellsy.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Add Your Sellsy App Info to Ampersand
+
+1. Log in to your [Ampersand Dashboard](https://dashboard.withampersand.com).
+
+2. Select the project where you want to create an Sellsy integration.
```
Comment (RajatPawar) [inline src/provider-guides/sellsy.mdx:54]:
```suggestion
2. Select the project in which you want to create a Sellsy integration.
```

File: src/provider-guides/sellsy.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+3. Select **Provider Apps**.
+
+4. Select _Sellsy_ from the **Provider** list.
+
+5. Enter the previously obtained _Client ID_ in the **Client ID** field and the _Client Secret_ in the **Client Secret** field.
+
+   ![Ampersand Integration](/images/provider-guides/sellsy.gif)
+
+6. Click **Save Changes**.
```
Comment (RajatPawar) [inline src/provider-guides/sellsy.mdx:66]:
```suggestion
```

**Review by RajatPawar — CHANGES_REQUESTED** [review]
Nice! Couple of minor changes required. After you are done, can you please make sure that all text in this guide has the words `a Sellsy ..` and not `an Sellsy`? Also, I'd suggest running this entire text through something like ChatGPT to fix grammatical errors / phrasing errors which should be the cherry on the top :)

File: src/provider-guides/sellsy.mdx (hunk 1 above)
Comment (sanjay-a01) [reply src/provider-guides/sellsy.mdx:40]: In the second version of the API, they switched to OAuth 2.0 for authentication. When creating an OAuth application, there will be an option named API V2 (beta).

image

File: src/provider-guides/sellsy.mdx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### Creating a Sellsy App
+
+Follow the steps below to create & configure a Sellsy app.
+
+1. Log in to your [Sellsy](https://login.sellsy.com/login) account.
+
+2. Go to **Menu**, **Settings**, and select **Developer Portal** in your Sellsy account.
+
+3. Click the **API V2 (beta)** tab, then click **Create API Access** and select your **API access type**: Private.
```
Comment (RajatPawar) [inline src/provider-guides/sellsy.mdx:38]:
```suggestion
3. Click the **API V2 (beta)** tab, then click **Create API Access** and select the **API access type** as *Private*.
```

File: src/provider-guides/sellsy.mdx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+   [Sellsy App Creation](/images/provider-guides/sellsy_OAuth.gif)
+
+7. Make sure to click **Save**.
+
+## Add Your Sellsy App Info to Ampersand
+
+1. Log in to your [Ampersand Dashboard](https://dashboard.withampersand.com).
+
+2. Select the project you want to create a Sellsy integration.
```
Comment (RajatPawar) [inline src/provider-guides/sellsy.mdx:54]:
```suggestion
2. Select the project you want to create a Sellsy integration in.
```

File: src/provider-guides/sellsy.mdx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+Follow the steps below to create & configure a Sellsy app.
+
+1. Log in to your [Sellsy](https://login.sellsy.com/login) account.
+
+2. Go to **Menu**, **Settings**, and select **Developer Portal** in your Sellsy account.
+
+3. Click the **API V2 (beta)** tab, then click **Create API Access** and select your **API access type**: Private.
+
+4. Enter the **Access name** for an API Access.
```
Comment (RajatPawar) [inline src/provider-guides/sellsy.mdx:40]:
```suggestion
4. Enter your application's name as the **Access name**.
```

**Review by RajatPawar — APPROVED** [review]

**Pull request merged**